packages/edge-config/src/index.ts

```diff
@@ -125,76 +128,103 @@ export function createClient(
   const loadersInstanceCache = new WeakMap<RequestContext, Loaders>();
 
   const api: Omit<EdgeConfigClient, 'connection'> = {
-    async get<T = EdgeConfigValue | undefined>(key: string): Promise<T> {
-      assertIsKey(key);
-      const loaders = getLoadersInstance(loaderOptions, loadersInstanceCache);
-
-      return loaders.get.load(key).then((value) => {
-        // prime has() with the result of get()
-        loaders.has.prime(key, value !== undefined);
-        return clone(value);
-      }) as Promise<T>;
-    },
-    async has(key): Promise<boolean> {
-      assertIsKey(key);
-      const loaders = getLoadersInstance(loaderOptions, loadersInstanceCache);
+    get: trace(
+      async function get<T = EdgeConfigValue | undefined>(
+        key: string,
+      ): Promise<T> {
+        assertIsKey(key);
+        const loaders = getLoadersInstance(loaderOptions, loadersInstanceCache);
 
-      return loaders.has.load(key);
-    },
-    async getMany<T = (EdgeConfigValue | undefined)[]>(
-      keys: string[],
-    ): Promise<T> {
-      const loaders = getLoadersInstance(loaderOptions, loadersInstanceCache);
+        return loaders.get.load(key).then((value) => {
+          // prime has() with the result of get()
+          loaders.has.prime(key, value !== undefined);
+          return clone(value);
+        }) as Promise<T>;
+      },
+      {
+        name: 'get',
```

Comment:
Does `key` need to be one of the attributes?

Reply:
That's your parameter above :). Sometimes it's convenient to add simple parameters as attributes to ease debugging.

Reply:
Ah right, yup, makes sense 👍